java/RSPEC-1862.java

```diff
@@ -0,0 +1,11 @@
+class ElifRepeatedCondition {
```

Comment:
RSPEC-2176: Class has same name as parent
<!-- lookout footnote separator -->
_Comment made by the analyzer sonarcheck_analyzer._